examples/docker-compose/README.md

```diff
@@ -5,13 +5,30 @@ docker containers using the cilium docker image.
 
 ## Requirements
 
+The easiest way is to use the [vagrant box](../../doc/vagrant.md) and just
+install docker-compose >=1.7.1 in it.
+
+If you want to install the dependencies manually, you need:
  - docker-compose (>=1.7.1)
- - Linux-kernel (>=4.8.0)
```

Comment:
Should this file be updated as well? https://github.com/cilium/cilium/blame/master/README.md#L95